helpers/github.ts

```diff
@@ -389,14 +391,14 @@ export async function createDevPoolIssue(projectIssue: GitHubIssue, projectUrl:
   // if the project issue is assigned to someone, then skip it
   if (projectIssue.assignee) return;
 
-  // if issue doesn't have the "Price" label then skip it, no need to pollute repo with draft issues
-  if (!(projectIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.PRICE))) return;
+  // if the issue has no price labels, it will be added to RFC repo
+  const hasNoPriceLabels = !(projectIssue.labels as GitHubLabel[]).some((label) => label.name.includes(LABELS.PRICE));
```

Comment:
With this approach we will [post tweets](https://github.com/ubiquity/devpool-directory/blob/20d74be226e8cae755e44d70d303d5d84acc49c8/helpers/github.ts#L416) for RFC kind issues which should not be the case as far as I understand

Reply:
fixed